tests/system/test_dbapi.py

```diff
@@ -691,32 +694,6 @@ def test_rollback_on_connection_closing(self, shared_instance, dbapi_database):
         cursor.close()
         conn.close()
 
-    def test_results_checksum(self):
```

Comment:
I assume that this test is being removed because it was wrong, but would you mind explaining what was wrong with it?

Reply:
cursor doesn't have a _checksum field now. Also this is more of a unit test which is already added